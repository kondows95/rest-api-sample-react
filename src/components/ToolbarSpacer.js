import React from 'react'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles(theme => ({
  toolbar: theme.mixins.toolbar,
}))

const ToolbarSpacer = () => {
  const classes = useStyles();
  return <div className={classes.toolbar} />
}

export default ToolbarSpacer
