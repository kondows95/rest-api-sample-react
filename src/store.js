import rootReducer from './modules/index';
import thunk from 'redux-thunk';
import { createStore, applyMiddleware } from 'redux';
export default createStore(rootReducer, applyMiddleware(thunk));