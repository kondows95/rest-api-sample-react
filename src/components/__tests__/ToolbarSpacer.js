import React from 'react'
import ReactDOM from 'react-dom';
import { act } from 'react-dom/test-utils';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'
import store from '../../store';
import ToolbarSpacer from '../ToolbarSpacer';
let container
beforeEach(() => {
  container = document.createElement('div')
  document.body.appendChild(container)
})

afterEach(() => {
  document.body.outerHTML = '<body></body>'
  container = null
})

describe("Confirm Toolbar component", () => {
  it('matches the snapshot', () => {
    const ToolbarSnapshot = renderer.create(
      <Provider store={store}>
        <Router>
          <ToolbarSpacer/>
        </Router>
      </Provider>
    ).toJSON();
    expect(ToolbarSnapshot).toMatchSnapshot();
  });
});