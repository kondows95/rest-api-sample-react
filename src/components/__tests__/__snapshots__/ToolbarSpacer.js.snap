// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm Toolbar component matches the snapshot 1`] = `
<div
  className="makeStyles-toolbar-1"
/>
`;
